testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/system/system_capabilities.t.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/**
+ * Tests system capabilities CRUD.
+ */
+StartTest(function(t) {
+  var waitForStoreToLoad = {
+    waitFor: 'storesToLoad', args: function() {
+        return t.cq1('nx-coreui-capability-list').getStore();
+    }
+  },
+      // Name of Capability tested
+      name = 'Base URL: Force';
+
+  t.describe('Given an admin when the admin is administering Capabilities', function(t) {
+    t.it('then CRUD should work', function(t) {
```

Comment:
Agree with you, we were following the functional test for tasks which does not.  (https://github.com/sonatype/nexus-oss/blob/master/testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/system/system_tasks.t.js) it would be nice to clean this one up as it off from the pattern you describe.